docs/pages/guides/quick-start.md

```diff
@@ -0,0 +1,297 @@
+        "encoder": {
+          "type": "ffmpeg_h264",
+          "preset": "ultrafast"
+        },
+        "initial": {
+          "root": {
+            "type": "view",
+            "background_color_rgba": "#4d4d4dff"
+          }
+        }
+      },
+      "audio": {
+        "encoder": {
+          "type": "opus",
+          "channels": "stereo"
+        },
+        "initial": {
+          "inputs": []
+        }
+      }
+    }
+    ```
+    You can configure output framerate and sample rate using [`LIVE_COMPOSITOR_OUTPUT_FRAMERATE`](../deployment/configuration.md#live_compositor_output_framerate) and [`LIVE_COMPOSITOR_OUTPUT_SAMPLE_RATE`](../deployment/configuration.md#live_compositor_output_sample_rate) environmental variables.
+
+    After receiving the response you can establish the connection and start listening for the stream. Check out [how to receive output streams](./receive-output.md) to learn more.
+
+    In this example we are using RTP over TCP, if you prefer to use UDP you need start listening on the specified port before sending register request to make sure you are not losing
+    first frames.
+  </TabItem>
+  <TabItem value="membrane" label="Membrane Framework">
+    ```elixir
+    def handle_init(ctx, opts) do
+      spec = [
+        ...
+
+        get_child(:live_compositor),
+        |> via_out(Pad.ref(:video_output, "video_output_1"), options: [
+          width: 1280,
+          height: 720,
+          encoder: %LiveCompositor.Encoder.FFmpegH264{
+            preset: :ultrafast
+          },
+          initial: %{
+            root: %{
+              type: :view,
+              background_color_rgba: "#4d4d4dff",
+            },
+          }
+        ])
+        |> video_output_1_spec,
+        
+        get_child(:live_compositor)
+        |> via_out(Pad.ref(:audio_output, "audio_output_1"), options: [
+          encoder: LiveCompositor.Encoder.Opus.t(),
+          initial: %{
+            inputs: []
+          }
+        ])
+        |> audio_output_1_spec
+
+        ...
+      ]
+      {[spec: spec], %{}}
+    end
+    ```
+    where `video_output_1_spec` and `audio_output_1_spec` are elements that can consume H264 video and Opus audio respectively.
+
+    You can configure output framerate and sample rate using [`framerate` and `output_sample_rate` bin options](https://hexdocs.pm/membrane_live_compositor_plugin/Membrane.LiveCompositor.html#module-bin-options).
+  </TabItem>
+</Tabs>
+
+`View` component does not have any children, so on the output you should see just a blank screen of a specified color as shown below.
+`inputs` list is empty, so the output audio will be silent.
+
+<div style={{textAlign: 'center'}}>
+    <img src={QuickStartEmpty} style={{ width: 600 }} />
+    Output stream
+</div>
+
+
+## Update output
+
+Configure it to:
+- show `input_1` and `input_2` [`InputSteams`](../api/components/InputStream.md) using [`Tiles`](../api/components/Tiles.md) component
```

Comment:
```suggestion
- Show input streams `input_1` and `input_2`  using [`Tiles`](../api/components/Tiles.md) component.
```
You linked to the input stream component, but you are referring to input stream itself, not the element of the scene. Also, it looks a bit wierd on the website.